widget_driver/README.md

```diff
@@ -52,6 +52,17 @@ Just focus on the current widget under test, and forget about all other child wi
 
 Let's get started and create our first `WidgetDriver`!
 
+### 1: Add a dependency to WidgetDriver
```

Comment:
there is already a section starting with `"1:"` below. Should we maybe change numeration?

Reply:
Thanks, I assumed they can all start with 1: and markup will fix the numbers for me, but that only works in lists.